app/models/account_migration.rb

```diff
@@ -113,6 +114,10 @@ def ephemeral_sender
     EphemeralUser.new(old_person.diaspora_handle, old_private_key)
   end
 
+  def validate_sender
+    raise if sender.nil?
```

Comment:
This `raise` looks weird. I was wondering why there isn't a message and then after looking twice I realized that `ephemeral_sender` will raise if sender couldn't be created. So I think just calling `sender` once and add a little comment that explains that calling `sender` will raise if needed is better I think.